.circleci/config.yml

```diff
@@ -938,25 +963,43 @@ workflows:
   launch-all-app-builds-workflow:
     # using do-builds param to select which of the two workflows using the build-on-tag-filters
     # should run; should be this one that just launches the builds or the one actually doing the builds
-    unless: << pipeline.parameters.do-builds >>
+    when:
+      and:
+        - not: << pipeline.parameters.do-builds >>
+        - equal: [ UNKNOWN, << pipeline.parameters.api_call >> ]
```

Comment:
New conditions to fix an existing bug - calling `.run_ci.sh deploy` also triggers `launch-all-app-builds-workflow`. It should not happen.

<img width="752" alt="Screenshot 2023-03-01 at 9 02 04 AM" src="https://user-images.githubusercontent.com/35533885/222251204-b0ac8b4b-0bb5-4bda-8c83-4ab37aa94793.png">